.chloggen/mx-psi_min-max-metrics1.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation 
+
+# The name of the component (e.g. pkg/quantile)
+component: pkg/otlp/metrics
+
+# A brief description of the change. Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate metrics.WithCountSumMetrics translator option.
```

Comment:
May be worth mentioning that `WithCountSumMetrics` will also send min and max (if present) moving forward.

Reply:
Makes sense, done!